testdata/image/gitserver/gitserver-ephemeral.yaml

```diff
@@ -25,7 +25,7 @@ items:
         serviceAccountName: git
         containers:
         - name: git
-          image: quay.io/openshifttest/origin-gitserver@sha256:8062457c330f0da521b340f2dceb13c4852d46ec58e45a9ef276a5ec639a328c
+          image: quay.io/openshiftteststaging/origin-gitserver:multiarch
```

Comment:
Hello @jeffdyoung, we need to use the SHA digest for the images. 
The reason for using digest is related to restricted network scenarios: in those cases, we use a mirror registry and using tags is not supported.

You can keep it with the tag until we need to merge (and the image is pushed on the openshifttest org).

Reply:
@aleskandro Makes sense, I'll change to sha after I run the tests on jenkins.  Do I open another PR to change `openshiftteststaging` to `openshifttest`, or does that happen after the PR is approved?